src/commands.rs

```diff
@@ -4,11 +4,13 @@
 pub mod bindle;
 /// Commands for building Spin applications.
 pub mod build;
-/// Command for deploying a Spin app to Hippo
+/// Commands for publishing applications to the Fermyon Platform.
```

Comment:
Super nit: Throughout you keep referring to the "Fermyon Platform" and I just wanted to make sure that this is our preferred terminology vs "Fermyon Cloud Platform".

Reply:
There are two distinct things, Fermyon Cloud and Fermyon Platform.  These commands work with both.  A new "Fermyon Cloud Platform" term would only add to the confusion I fear!

Reply:
I believe the current verbiage is the correct convention.

My understanding is that we use "Fermyon Platform" to refer to the self-hosted as well as the managed service (Fermyon Cloud). We refer to both the self-hosted and the hosted option on [fermyon.dev](https://www.fermyon.dev/).

Reply:
Cool, thanks for the thorough response you two 😄 . Looks good.